upstream-community-operators/iot-simulator/iot-simulator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,398 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: iot-simulator.v0.1.0
+  namespace: olm
+  annotations:
+    categories: "Streaming & Messaging"
+    certified: "false"
+    description: An IoT device simulator, simulating producers and consumers using the Eclipse Hono API
+    containerImage: ctron/iot-simulator-operator:0.1.0
+    createdAt: 2019-03-19T00:00:00Z
+    support: none
+    capabilities: Seamless Upgrades
+    repository: https://github.com/ctron/iot-simulator-operator
+    alm-examples: |-
+      [{
+          "apiVersion": "iot.dentrassi.de/v1alpha1",
+          "kind": "Simulator",
+          "metadata": {
+            "name": "hono1",
+              "namespace": "iot-simulator"
+          },
+          "spec": {
+              "endpoint": {
+                  "messaging": {
+                      "user": "foo",
+                      "password": "bar",
+                      "host": "messaging.host.hono.svc",
+                      "port": "5671",
+                      "caCertificate": ""
+                  },
+                  "registry": {
+                      "url": "https://your-device-registry"
+                  },
+                  "adapters": {
+                      "mqtt": {
+                          "host": "mqtt-adapter.hono.svc",
+                          "port": 8883
+                      },
+                      "http": {
+                          "url": "https://http-adapter.hono.svc"
+                      }
+                  }
+              }
+          }
+      },
+      {
+        "apiVersion": "iot.dentrassi.de/v1alpha1",
+        "kind": "SimulatorConsumer",
+        "metadata": {
+          "name": "consumer1",
+          "namespace": "iot-simulator"
+        },
+        "spec": {
+          "simulator": "hono1",
+          "tenant": "myapp1.iot",
+          "type": "telemetry",
+          "replicas": 1
+        }
+      },
+      {
+        "apiVersion": "iot.dentrassi.de/v1alpha1",
+        "kind": "SimulatorProducer",
+        "metadata": {
+          "name": "producer1",
+          "namespace": "iot-simulator"
+        },
+        "spec": {
+          "simulator": "hono1",
+          "tenant": "myapp1.iot",
+          "type": "telemetry",
+          "protocol": "http",
+          "replicas": 1,
+          "numberOfDevices": 10
+        }
+      }
+      ]
+
+spec:
+  maturity: alpha
+  displayName: IoT simulator
+  description: |
+    A scalable IoT simulator, which simulates producers (MQTT or HTTP) and consumers.
+
+    ## Pre-requisites
+
```

Comment:
Could you also include the dependency on prometheus CRs here? This will require that the prometheus operator be installed via OLM.

Reply:
Ok, I assumed that it would be installed automatically, as it was defined in the "required" section. Added a note about that.